browser/components/EditGroup.tsx

```diff
@@ -20,17 +20,17 @@ import * as actions from '../actions/strategicActions';
 
 interface Props {
   params: any;
-  fetchGroup: any;
-  fetchGroups: any;
-  fetchProjects: any;
-  fetchUsers: any;
-  updateGroup: any;
-  deleteGroup: any;
+  fetchGroup: (id) => void;
+  fetchGroups: () => void;
+  fetchProjects: () => void;
+  fetchUsers: () => void;
+  updateGroup: (group) => void;
+  deleteGroup: (id) => void;
```

Comment:
Se my previous reply about creating better defined interfaces with optional fields.

Reply:
These specifically highlighted ones are all function pointers explaining the input and the output, but I will change it for the other json objects.